lib/grandpa/grandpa.go

```diff
@@ -83,15 +161,16 @@ func (s *Service) isCompletable() (bool, error) {
 	return true, nil
 }
 
-// getPreVotedBlock returns the current pre-voted block B.
+// getPreVotedBlock returns the current pre-voted block B. also known as GRANDPA-GHOST.
 // the pre-voted block is the block with the highest block number in the set of all the blocks with
 // total votes >= 2/3 the total number of voters, where the total votes is determined by getTotalVotesForBlock.
 func (s *Service) getPreVotedBlock() (Vote, error) {
-	blocks, err := s.getPossiblePreVotedBlocks()
+	blocks, err := s.getPossibleSelectedBlocks(prevote)
```

Comment:
Is the naming based on spec/substrate? If not, maybe simplified to `getBlockCandidates()`.

Reply:
...but it looks like `getPossibleSelectedBlocks` makes more sense with `getPossibleSelectedAncestors`

Reply:
this function isn't based on the spec, yeah the naming is a bit weird. I like using `candidates`, I'll consider what a better name would be